ocw/lib/emailnotify.py

```diff
@@ -33,28 +33,23 @@ def draw_instance_table(objects):
 
 
 def send_leftover_notification():
-    cfg = ConfigFile()
-    if not cfg.has('notify'):
+    if not ConfigFile().has('notify'):
         return
     o = Instance.objects
     o = o.filter(active=True,
                  csp_info__icontains='openqa_created_by',
-                 age__gt=timedelta(hours=int(cfg.get(['notify', 'age-hours'], 12))))
+                 age__gt=timedelta(hours=PCWConfig().get_feature_property('notify', 'age-hours')))
 
     if o.filter(notified=False).count() == 0:
         return
 
-    subject = cfg.get(['notify', 'subject'], 'CSP left overs')
+    subject = PCWConfig().get_feature_property('notify', 'subject')
     body_prefix = "Message from {url}\n\n".format(url=build_absolute_uri())
     send_mail(subject, body_prefix + draw_instance_table(o))
 
     # Handle namespaces
-    namespaces = list(dict.fromkeys([i.vault_namespace for i in o]))
-    for namespace in namespaces:
-        cfg_path = ['notify.namespace.{}'.format(namespace), 'to']
-        if not cfg.has(cfg_path):
-            continue
-        receiver_email = cfg.get(cfg_path)
+    for namespace in PCWConfig.get_namespaces_for('notify'):
```

Comment:
@cfconrad  there is some change in the flow . WDYT ?

Reply:
The result looks similar to me.
The only thing which is missing is:
```
if not receiver_email:
   continue
```
or?

Reply:
I would call this code duplication . `send_mail` have handling for `reciever_email=None`